python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -27,6 +27,11 @@ def getInfo(self, filename):
                 info['calibDate'] = date
         return phuInfo, infoList
 
+    def _translateFromCalibId(self, field, md):
```

Comment:
Agree. Now the design is the same as `obs_subaru`, and hopefully will be improved in the future.

DM-7286 and DM-7287 will help regarding tests.